website/admin/views/__init__.py

```diff
@@ -44,3 +37,33 @@ def logout():
     utilities.session.logout()
     flash('Successfully logged out', 'success')
     return redirect(url_for('public.index'))
+
+
+@admin_site.route('/triggers')
+def triggers():
+    return render_template("admin/admin_triggers.html")
+
+
+@admin_site.route('/help')
+def help():
+    return "admin help to be implemented"
+
+
+
+@admin_site.route('/account/settings')
+def account_settings():
+    return "To be implemented"
+
+
+status = ['Enabled', 'Disabled']
+themeinfo = [{'id': '1', 'name': 'Weekend Away Theme', 'theme_status': status[0]},
+             {'id': '2', 'name': 'Night Party Theme', 'theme_status': status[1]}]
+
+
+@admin_site.route('/themes')
```

Comment:
You probably need the user that is being seen by the admin as a parameter. Or how would you handle this?